src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -332,10 +331,13 @@ public static String abbreviate(final String str, final String abbrevMarker, fin
      * @since 3.6
      */
     public static String abbreviate(final String str, final String abbrevMarker, int offset, final int maxWidth) {
-        if (isEmpty(str) || isEmpty(abbrevMarker)) {
+        if (isEmpty(str) && isEmpty(abbrevMarker)) {
+            return str;
+        } else if (isNotEmpty(str) && "".equals(abbrevMarker) && maxWidth>0) {
```

Comment:
No magic strings, this class defines `EMPTY` for this purpose. You are missing spaces around `>`

Reply:
I added the spaces. What are you implying about the magic strings? Forgive me, I'm not as experienced of a programmer.

Reply:
A "magic" string or number is a value that is hard-coded, as opposed to reusing a constant, in this case, `EMPTY` instead of `""`.

Reply:
Thanks for the explanation, I removed the magic string